Message.php

```diff
@@ -233,7 +234,22 @@ protected function setBody($body, $contentType)
             }
         }
     }
-
+    
+    /**
+     * Adds a Swift_Signers_DKIMSigners object to the Swift_SignedMessage message.
+     * This requires the messageClass to be set to \Swift_SignedMessage instead of \Swift_Message.
+     * messageClass can be set to the correct value via yii\swiftmailer\Mailer::$messageConfig
+     * @param string $privateKeyAlias an alias pointing to the private key file.
+     * @param string $domain the domain to sign with.
+     * @param string $selector the dkim domain selector.
+     */
+     public function setDkim($privateKeyAlias, $domain, $selector) {
+        $privateKey = file_get_contents(Yii::getAlias($privateKeyAlias));
+        $dkimSigner = new \Swift_Signers_DKIMSigner($privateKey, $domain, $selector);
+        $this->getSwiftMessage()->attachSigner($dkimSigner);
```

Comment:
Because it no longer requires a different Swift Message class. They merged it here, which has been released a while back (5.0.3 i think).

https://github.com/swiftmailer/swiftmailer/commit/a64a47a4c325085e7cbca1b6151d8885e0c28b2d

Thats why I changed the swiftmailer composer requirements. However I see I made a mistake there of unnecessarily jumping to the next minor.